project/frontend/src/index.js

```diff
@@ -13,11 +13,30 @@ import HomePage from './pages/HomePage.js';
 import MatchResultsPage from './pages/MatchResultsPage.js';
 
 import RestaurantCard from './components/RestaurantCard.js';
+import RestaurantCardDeck from './components/RestaurantCardDeck.js';
+import RestaurantCardStack from './components/RestaurantCardStack.js';
 
 import * as serviceWorker from './serviceWorker';
 
 // TODO: remove
-import { restaurant, details } from './components/SampleRestaurant.js';
+import generateRestaurant from './components/SampleRestaurant.js';
+const numberOfCardsToGenerate = 20;
+const cards = [];
+for (let i = 0; i < numberOfCardsToGenerate; i++) {
+  const data = generateRestaurant(
+    'Amarena ' + i,
+    'apr75h4bni2pf98h4inujnksjrliu34' + i
+  );
+  if (i === 3) {
+    data.restaurant.key.name += ' Of the California State of the US';
+  } else if (i === 4) {
+    data.restaurant.key.address = '123 Main Street';
+  } else if (i === 7) {
+    data.details.result.website +=
+      '?somethingextra=123&antherthing=1234567890&type=json';
+  }
```

Comment:
does this logic mean that we see the address peeking out for card # 4 etc ? (I think this block could use comments explaining that -- the 3,4,7 seems arbitrary without explanation)

Reply:
This stuff was really just for testing purposes. I wanted to have the cards render with different collapsible div heights, so I changed the text content of some attributes. This will be removed in a future PR, if it was not already removed in #32.